ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -392,51 +356,42 @@ private void createBanner() {
       return;
     }
 
-    cleanUp();
```

Comment:
looks like this call was for VungleBannerAdapter passed from one VungleInterstitialAdapter to another.
In this implementation it's impossible
We can finish another one when getBannerRequest allows, but don't use it